src/app/games/trexo/TrexoTutorial.ts

```diff
@@ -0,0 +1,74 @@
+    movesAndStates.push({ move, state });
+    previousState = state;
+}
+const firstMove: TrexoMove = TrexoMove.from(new Coord(5, 5), new Coord(5, 4)).get();
+const stateAtTurnOne: TrexoState = TrexoRules.get().applyLegalMove(firstMove, initialState);
+const secondMove: TrexoMove = TrexoMove.from(new Coord(5, 7), new Coord(5, 6)).get();
+const stateAtTurnTwo: TrexoState = TrexoRules.get().applyLegalMove(secondMove, stateAtTurnOne);
+
+export class TrexoTutorial extends Tutorial {
+    public tutorial: TutorialStep[] = [
+        // 1. but du jeu et plateau
+        TutorialStep.informational(
+            $localize`Goal of the game`,
+            $localize`At Trexo, the goal of the game is to align 5 piece of your color in an horizontal, vertical or diagonal line. But player must put a tile on the board, and that tile is constituted of two pieces, one from each players!`,
+            TrexoState.getInitialState()),
+        TutorialStep.anyMove(
+            $localize`Dropping a tile`,
+            $localize`When you drop at tile, it need to be on even ground, and it cannot be right on top of another tile. In others words, it need to be either on the floor, either on two tiles at the same height. To do that, just click on the place where you want to put the opponent piece, then on the neighboring square where you want to put your piece.<br/><br/>You're playing Dark, go ahead.`,
+            TrexoState.getInitialState(),
+            TrexoMove.from(new Coord(4, 4), new Coord(3, 4)).get(),
+            $localize`Congratulations!`),
+        TutorialStep.fromMove(
+            $localize`Put a piece over other pieces`,
+            $localize`When two different pieces on the same level touch each other, you can put a piece on them. You must respect two rules for that:<br/><ul><li>The two pieces must be at the same level.</li><li>You cannot drop your piece on only one other piece.</li></ul><br/>You're playing Dark, do such a move!`,
+            stateAtTurnTwo,
+            [
+                TrexoMove.from(new Coord(5, 5), new Coord(5, 6)).get(),
+                TrexoMove.from(new Coord(5, 6), new Coord(5, 5)).get(),
+            ],
+            $localize`Congratulations!`,
+            $localize`Incorrect, you put that piece on the floor level, please put it on top of two pieces.`)
+            .withPreviousMoveAndState([
+                { move: firstMove, state: initialState },
+                { move: secondMove, state: stateAtTurnOne },
+            ]),
+        TutorialStep.fromPredicate(
+            $localize`Victory`,
+            $localize`To win at Trexo, you need to align 5 of your pieces, only the piece on top count. Since you have to put the piece of your opponents first, if you create a victory for them and for you in the same turn, they'll be considered the winners (if you create only a victory for them too, obviously).<br/><br/>You're playing Dark, win.`,
```

Comment:
il y a qu'un seul opponent !
J'aurais coupé la première phrase à la première virgule, ou reformulé.
"The first player that has 5 of its visible faces aligned wins" par exemple
Aussi, peut-être éviter de parler de "piece", mais plutôt "tile" et "side/face" ?

Reply:
J'suis pas sur que face soit clair, genre, tu me dit la face de la tuile j'imagine un truc bicolore.
Peut juste dire "le premier à aligner cinq de ses pièces gagnes sauf si un alignement adverse est crée, auquel cas le joueur perd" ?

Reply:
"Le premier à aligner cinq de ses pièces gagne. Si deux alignements se créent en même temps, le dernier joueur perd" ?

Reply:
"Le premier à aligner cinq de ses pièces gagnes. Si vous créez un alignement adverse [ou un de chaque?], vous perdez" ?

Reply:
oui !